### DISCUSSION TITLE
Want to use ProductSwitcher icon but paste version in console is not up to date

### DISCUSSION THREAD

**Discussion by uditadhikari117** [twilio-labs/paste#3390] [General] [answered]
Hi folks,
I have been requested to use **ProductSwitcherIcon**. However, the console zen does not have latest Paste and upgrading that is outside the scope of this ticket.

Is there a way for me to manually download this icon and use it within my component?

**Comment by SiTaggart** [comment, marked as answer]
Hi @uditadhikari117 

You can grab the icon directly from source https://github.com/twilio-labs/paste/tree/main/packages/paste-icons/svg and copy and paste the svg.

We **absolutely do not recommend** doing this. Not only is it duplicative work, because we've already built the [Product Switcher](https://paste.twilio.design/components/product-switcher), it's pure debt that potentially adds something that could further block any updates to the Paste dependency in the future.

**Comment by uditadhikari117** [comment]
I rather use Product Switcher that has been built. Is there a way for me to use it without upgrading paste?

**Reply by TheSisb** [reply]
There may be ways to do so, but to re-iterate @SiTaggart, "it's pure debt that potentially adds something that could further block any updates to the Paste dependency in the future." so we do not advise it as a path forward.

**Reply by uditadhikari117** [reply]
Right but there is a deadline to deliver this asap and in that time we cannot upgrade paste. What is the way to use it without paste upgrade?

**Reply by SiTaggart** [reply]
You can't "use it" as such, you have to recreate it using the stuff that we use. All the source code is open source.

Again, strongly recommend you should not do this, though. You should mark it as a blocker on your project.
